index.html

```diff
@@ -0,0 +1,133 @@
+  <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap" rel="stylesheet">
+  <script
+      src="https://kit.fontawesome.com/e520ead38a.js"
+      crossorigin="anonymous"
+    ></script>
+</head>
+<body>
+  <header>
+    <nav>
+      <div class="top">
+        <i class="fa-brands fa-facebook-f social"></i>
+        <i class="fa-brands fa-twitter social"></i>
+        <ul class="top-list">
+          <li class="top-items">English</li>
+          <li class="top-items">MyPage</li>
+          <li class="top-items">Logout</li>
+        </ul>
+      </div>
+      <div class="main-nav">
+        <img src="./images/microverseLogo.png" class="logo" alt="Microverse Logo">
+        <ul class="menu-mobile" id="menu">
+          <li class="mobile-menu-list" onclick="closeMenu()"><a href="./about.html">About</a></li>
+          <li class="mobile-menu-list" onclick="closeMenu()"><a href="#">Program</a></li>
+          <li class="mobile-menu-list" onclick="closeMenu()"><a href="#">Join</a></li>
+          <li class="mobile-menu-list" onclick="closeMenu()"><a href="#">Sponsor</a></li>
+          <li class="mobile-menu-list" onclick="closeMenu()"><a href="#">News</a></li>
+          <i class="fa-solid fa-xmark close" onclick="closeMenu()"></i>
+        </ul>
+        <ul class="menu">
+          <li class="menu-list"><a href="./about.html">About</a></li>
+          <li class="menu-list"><a href="#">Program</a></li>
+          <li class="menu-list"><a href="#">Join</a></li>
+          <li class="menu-list"><a href="#">Sponsor</a></li>
+          <li class="menu-list"><a href="#">News</a></li>
+          <button type="button" class="button">CC Campaign</button>
+        </ul>
+      </div>
+      <i class="fa-solid fa-bars open" onclick="openMenu()"></i>
+    </nav>
+  </header>
+  <main>
+    <section class="body">
+      <article class="main-text">
+        <p class="intro">"Hello! Sharing world"</p>
+        <h1 class="title">Become an international software developer, from anywhere.</h1>
+        <article class="information">
+          <p>Gain the skills you need to get hired. Learn online, connect to life-changing opportunities. Pay $0 until you land a job!</p>
+        </article>
+      </article>
+    </section>
+
+<!-- -----------------------------main program-------------------------------- -->
+
+    <section class="program">
+      <div class="program-container">
+        <h2 class="main-program">Main Program</h2>
+        <div class="line"></div>
+        <div class="method-container">
+          <div class="method"> 
+            <i class="fa-solid fa-person-chalkboard method-icon"></i>
+            <h3 class="method-title">Lecture</h3>
+            <p class="method-info">Lorem ipsum dolor sit amet consectetur, adipisicing elit. Atque sapiente nesciunt dolores!</p>
+          </div>
```

Comment:
- [ ] Kindly use an `<article>` tag instead of a `<div>` as it is a self-contained composition on the page. It is better for the semantic structure of the page. Please refer to this [MDN documentation](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article) to get more information about when to use `article` tag. 😎️

Reply:
Fixed.